.github/workflows/superset-backend.yml

```diff
@@ -0,0 +1,161 @@
+name: Backend
+
+on:
+  # only build on direct push to `master` branch
+  push:
+    branches: [ master ]
+    paths:
+    - superset/**/*.py
+    - tests/**
+    - setup.py
+    - requirements*.txt
+  # but also build on pull requests to any branch
+  # (the so-called feature branch)
+  pull_request:
+    paths:
+    - superset/**/*.py
+    - tests/**
+    - setup.py
+    - requirements*.txt
+
+jobs:
+  python-lint:
+    runs-on: ubuntu-latest
```

Comment:
Wondering if we should pin this to `ubuntu-18.04` and bump periodically as new versions arrive?

Reply:
All GitHub's official docs used `ubuntu-latest`, but I agree it's probably better to pin the version just to be safe.